assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/factory.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```

Comment:
does it make sense to have a single factory which will take framework and task as discriminator?

Reply:
Updated.